rules/sflow.mk

```diff
@@ -1,6 +1,6 @@
 # host-sflow package
 
-HSFLOWD_VERSION = 2.0.35
```

Comment:
Any reason for not using latest version 2.0.51 ? or stable previous versions such as 2.0.49?

Reply:
No specific reason. Egress sample support was added in 2.0.45. Hence updated to this version.